src/App.js

```diff
@@ -1,26 +1,10 @@
 import React from 'react';
-import logo from './logo.svg';
 import './App.css';
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          <code>src/App.js</code>
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <h1 className="text-xl leading-6 font-bold text-gray-900">Calculator APP</h1>
     </div>
   );
 }
```

Comment:
- [ ] From the requirements of this project, we need to import the Calculator.js component into this App.js(main component). Please let's consider implementing this.